openedx/core/djangoapps/user_authn/templates/user_authn/edx_ace/passwordresetsuccess/email/subject.txt

```diff
@@ -0,0 +1,4 @@
+{% load i18n %}
+{% autoescape off %}
+{% blocktrans trimmed %}Your {{ platform_name }} password has changed{% endblocktrans %}
```

Comment:
The subject should be something like `Successful Password Reset` or `{platform} password reset`.

Reply:
Changed it to `Password Reset Success`